packages/blade/scripts/generateCSSTokens.js

```diff
@@ -0,0 +1,169 @@
+const { getLeaves } = require('any-leaf');
+const StyleDictionary = require('style-dictionary');
+const set = require('lodash/fp/set');
```

Comment:
I am using the [Function Programming version](https://github.com/lodash/lodash/wiki/FP-Guide) of `set` which does not mutate the original theme object. I spent quite some time figuring out how to avoid this mutation causing issues while generating other themes but couldn't really work it out. Non-mutative method is much more straightforward

Reply:
did you try using deepClone from lodash? that deep clones object and then you can use it regularly and it'll not have any mutations

Reply:
I'll check this

Reply:
Yup this works well!